### PR TITLE
Added Unit Test Case For PropertyUtil.java

### PR DESCRIPTION
**Pull request by Gowrishankar04**
Fixed : https://github.com/crowdin/android-studio-plugin/issues/84

### CODE REVIEW

File: src/test/java/com/crowdin/util/PropertyUtilTest.java [hunk 1]
```diff
@@ -1,76 +1,45 @@
...
 
-import static org.junit.Assert.assertEquals;
 import static org.junit.jupiter.params.provider.Arguments.arguments;
 
-public class PropertyUtilTest {
+public class PropertyUtilTest extends BasePlatformTestCase {
+
+    @Override
+    protected String getTestDataPath() {
+        return "src/test/testData";
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/util/PropertyUtilTest.java:17]: This test is not testing anything

File: src/test/java/com/crowdin/util/PropertyUtilTest.java [hunk 2]
```diff
@@ -1,76 +1,45 @@
...
-        CrowdinProperties result = CrowdinPropertiesLoader.load(properties);
-        assertEquals("Properties: " + properties, expected, result.getFiles());
+    public void testGetSources(String project, String expected, String key) {
+        myFixture.copyFileToProject(project);
+        String result = PropertyUtil.getPropertyValue(key, myFixture.getProject());
+        assertEquals(expected, result);
     }
 
     public static Stream<Arguments> testGetSources() {
-        ClassLoader classLoader = PropertyUtilTest.class.getClassLoader();
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/util/PropertyUtilTest.java:31]: Why the content of this test was deleted?

File: src/test/java/com/crowdin/util/PropertyUtilTest.java [hunk 3]
```diff
@@ -1,76 +1,45 @@
...
+    public void teardown() throws Exception {
+        super.tearDown();
+    }
 
     @ParameterizedTest
     @MethodSource
-    public void testGetSources(Properties properties, List<FileBean> expected) {
-        CrowdinProperties result = CrowdinPropertiesLoader.load(properties);
-        assertEquals("Properties: " + properties, expected, result.getFiles());
+    public void testGetSources(String project, String expected, String key) {
```
Comment (andrii-bodnar) [inline src/test/java/com/crowdin/util/PropertyUtilTest.java:32]: It's a duplicated test name. The names should be unique and describe what is being tested in this test

**Comment by andrii-bodnar** [comment]
@Gowrishankar04 thank you for your contribution! Please take a look at the comments I've left to this PR

**Comment by andrii-bodnar** [comment]
Hi @Gowrishankar04, any updates on this?

**Comment by Gowrishankar04** [comment]
Hi @andrii-bodnar , Sorry for the delay.
I am facing issue while creating a mock object for the type Project. Though, tried different ways, I am unable to create it. Could you please guide me on this?

Thanks

**Comment by andrii-bodnar** [comment]
Hi @Gowrishankar04

I guess we have to skip writing new tests for this part of allocation as probably we can't mock the Project class.

I should have noticed that before creating this issue, sorry

**Comment by andrii-bodnar** [comment]
@Gowrishankar04 you can write some Unit tests for some other part of the application if you wish

Of course, it will be considered as hacktoberfest contribution

**Pull request closed without merging**